src/data/roadmap.yml

```diff
@@ -108,7 +109,7 @@
 - tags: completed
   image: "roadmap/DEXCompleted.svg"
   title: "Proposal For a Decentralized Exchange"
-  date: "Added Q1 2018"
+  date: "Completed Q2 2018"
```

Comment:
DEX proposal was Q1 2019

https://proposals.decred.org/proposals/5431da8ff4eda8cdbf8f4f2e08566ffa573464b97ef6d6bae78e749f27800d3a

Reply:
You sure?
https://www.decred.org/press/ releases section
says June 5 2018

Maybe it's for something different?

Reply:
Yeah the proposal was the link I pasted above.

June 2018 was just a blog post announcing the proposal.